lib/Console/Command/Parse.php

```diff
@@ -311,7 +319,6 @@ protected function generateGlSection(\SimpleXMLElement $gl, OglVersion $glVersio
         $glsl->addAttribute("version", $glVersion->getGlslVersion());
 
         foreach ($glVersion->getExtensions() as $glExt) {
-            \Mesamatrix::$logger->debug('Processing extension '.$glExt->getName());
```

Comment:
Same here: I removed it because it's not really needed anymore since it's just basic XML processing, nothing wrong can happen.